src/DynamoCore/Graph/Workspaces/HomeWorkspaceModel.cs

```diff
@@ -750,8 +750,9 @@ private void OnUpdateGraphCompleted(AsyncTask task)
                 var node = workspace.Nodes.FirstOrDefault(n => n.GUID == guid);
                 if (node == null)
                     continue;
-
+                node.SuppressStatePropertyNotifications = true;
                 node.Warning(message.Value); // Update node warning message.
+                node.SuppressStatePropertyNotifications = false;
```

Comment:
if this solution proves to be the best way to solve this problem (generically).
We can create a smart object to make enabling/disabling easier
```
using(new SuppressStatePropertyNotifications(["prop1","prop2"])) {
code...
}
```

Reply:
RAII: https://www.stubbornprogrammer.com/2020/07/11/raii-in-c-2/